Added integration tests to appengine-push.

tmatsuo opened the pull request:
I'm going to add real integration tests to this PR.

jonparrott commented on appengine-push/test_deploy.py:
```diff
@@ -0,0 +1,81 @@
+
+    def fetch_messages(self):
+        """Fetch messages"""
+        conn = httplib.HTTPSConnection(get_gae_host())
```
why httplib instead of httplib2?

tmatsuo replied:
No strong readon, but httplib is in the standard lib and it satisfy my needs here.

jonparrott commented:
LGTM with small nit.

tmatsuo commented:
Thanks, I changed to httplib2 from httplib.
Also added some integration tests for checking consistency of base64 variants used on the server side and the client side.

@jonparrott PTAL

jonparrott commented:
LGTM

tmatsuo commented:
Thanks! will merge when the travis says OK :)
